src/components/opoTable/opoTable.js

```diff
@@ -14,10 +16,20 @@ export default function OpoTable({
   opos,
   title,
 }) {
-  if (!opos.length) return null;
 
+
+const columns = useMemo(() => {
+  const cols = [
+    "states",
+    "name",
+    "region",
+    "tier",
+    "donors",
+    "shadow",
+    "investigation",
+  ];
   const Heading = ({ color, title }) => (
-    <th scope="col" className={color ?? null}>
+    <div>
```

Comment:
nit: if we're not using `div` this can be `<></>`

Reply:
surprisingly, this div keeps the footnote next to the words, instead of with the chevron. when we remove the footnotes it won't matter anymore.